src/css/controls/flags/audioplayer.less

```diff
@@ -77,17 +77,18 @@
         }
 
         .jw-horizontal-volume-container {
+            transition: width 300ms @default-timing-function;
```

Comment:
@pajong time slider should also get this; it doesn't close with a transition

Reply:
I applied transition to the horizontal volume container when closing, so it has the same effect. Let me know if I am wrong.

Reply:
what i meant is that the time slider closes instantaneously, instead of closing with a transition. The transitions for time and volume sliders should be the same